packages/web/src/components/PaginatedTable/index.tsx

```diff
@@ -10,35 +10,52 @@ interface Props {
 }
 
 const PaginatedTable: React.FC<Props> = ({ request, children }) => {
-  const { data, response, hasPreviousPage, hasNextPage, loadPrevious, loadNext } = request;
+  const {
+    data,
+    response,
+    hasPreviousPage,
+    hasNextPage,
+    loadPrevious,
+    loadNext,
+  } = request;
 
   return (
     <>
-      { data && (
+      {data && (
         <Pagination>
           <p>
-            <span>{response?.headers['x-total-count']} registro(s)</span>
-            <span>{response?.headers['x-total-page']} página(s)</span>
+            <span>{response?.headers['x-total-count']}
```

Comment:
The identation here got a little messed up. Do you think that we can fix that or ESLint is blocking?

Reply:
An idea that came to my mind is maybe we could move  `response?.headers['x-total-count']` to a `numberOfRegister` variable and `response?.headers['x-total-page']` to `numberOfPages`.

Reply:
@eugustavo That would be great!